instrumentation/github.com/aws/aws-lambda-go/otellambda/wrapLambdaHandler.go

```diff
@@ -63,7 +63,7 @@ func validateReturns(handler reflect.Type) error {
 }
 
 // Wraps and calls customer lambda handler then unpacks response as necessary.
-func (whf *wrappedHandlerFunction) wrapperInternals(ctx context.Context, handlerFunc interface{}, eventJSON []byte, event reflect.Value, takesContext bool) (interface{}, error) {
+func (whf *wrappedHandlerFunction) wrapperInternals(ctx context.Context, handlerFunc interface{}, eventJSON []byte, event reflect.Value, takesContext bool) (interface{}, error) { // nolint:revive // this control flag is required
```

Comment:
This lint issue is surprising, and maybe a false positive?

> flag-parameter: parameter 'len' seems to be a control flag, avoid control coupling

There is no `len` parameter though.

Reply:
Looks like already fixed but not released: https://github.com/mgechev/revive/issues/1211